src/SFA.DAS.AssessorService.Application/Handlers/Staff/StartCertificateHandler.cs

```diff
@@ -89,12 +91,10 @@ public async Task<Certificate> Handle(StartCertificateRequest request, Cancellat
             return certificate;
         }
 
-        private async Task<Certificate> CreateNewCertificate(StartCertificateRequest request)
+        private async Task<Certificate> CreateNewCertificate(StartCertificateRequest request, Organisation organisation)
         {
             _logger.LogInformation("CreateNewCertificate Before Get Ilr from db");
             var ilr = await _ilrRepository.Get(request.Uln, request.StandardCode);
-            _logger.LogInformation("CreateNewCertificate Before Get Organisation from db");
-            var organisation = await _organisationQueryRepository.GetByUkPrn(request.UkPrn);
             _logger.LogInformation("CreateNewCertificate Before Get Provider from API");
             var provider = await GetProviderFromUkprn(ilr.UkPrn);
```

Comment:
New SV Code: In the case of multiple standard version why is only the title of the first recorded on line 155 but for a single version all the properties are recorded on lines 137-148?

When a deleted certificate is re-used and in the mean time there may be additional standard versions should these versions which are stored in the certificate data not get refreshed?

Reply:
This is because when there are multiple versions the title is known but the other details will be supplied from the UI